Externals_CAM.cfg

```
@@ -22,8 +22,8 @@ required = False
 [ncar-physics]
 local_path = src/physics/ncar_ccpp
 protocol = git
-repo_url = https://github.com/NCAR/atmospheric_physics
-tag = atmos_phys0_00_018
+repo_url = https://github.com/K20shores/atmospheric_physics
+branch = musica_ccpp
```

Typically we do not allow personal branches into CAM and I'm guessing the same holds for CAM-SIMA.  Of course y'all may be on top of this already

---

I'm not sure what's proper protocol here. Musica is not yet merged into the atmospheric physics repo. Once it is, we can change the URL here

---

That PR is [here](https://github.com/NCAR/atmospheric_physics/pull/63), just to establish a link in github

---

I agree that we can resolve this concern by simply getting the atmospheric_physics PR merged and tagged first before merging in these changes.  

Also, while it would be good to have the new MICM variables in the standard names dictionary soon, we can always get everything else merged in first and then always go back and update the names later in a separate PR if NOAA requests any changes.

---

@K20shores A new tag now exists for the `atmospheric_physics` mods you added, so I believe you can replace the current code changes with:

```suggestion
repo_url = https://github.com/NCAR/atmospheric_physics
tag = atmos_phys0_00_019
```

Which should resolve @cacraigucar's concern here.